packages/web/src/components/comments/TimestampLink.tsx

```diff
@@ -1,24 +1,60 @@
 import { useCurrentCommentSection } from '@audius/common/context'
-import { seek } from '@audius/common/src/store/player/slice'
-import { TextLink, formatTrackTimestamp } from '@audius/harmony'
-import { useDispatch } from 'react-redux'
+import { useGatedContentAccess } from '@audius/common/hooks'
+import { ModalSource } from '@audius/common/models'
+import {
+  playerActions,
+  PurchaseableContentType,
+  trackPageLineupActions,
+  trackPageSelectors,
+  usePremiumContentPurchaseModal
+} from '@audius/common/store'
+import { formatTrackTimestamp, TextLink, TextLinkProps } from '@audius/harmony'
+import { useDispatch, useSelector } from 'react-redux'
+
+const { tracksActions } = trackPageLineupActions
+const { getLineup } = trackPageSelectors
+const { seek } = playerActions
+
+type TimestampLinkProps = {
+  timestampSeconds: number
+} & TextLinkProps
+
+export const TimestampLink = (props: TimestampLinkProps) => {
+  const { timestampSeconds, ...other } = props
 
-export const TimestampLink = ({
-  trackTimestampS
-}: {
-  trackTimestampS: number
-}) => {
   const dispatch = useDispatch()
-  const { playTrack } = useCurrentCommentSection()
-  const handleClick = () => {
-    // Starts playing the current page's song
-    playTrack()
```

Comment:
Since we're not using this anymore, can you remove this from the context?

Reply:
Oh right I was thinking about that and then forgot